eng/Packages.Data.props

```diff
@@ -211,14 +213,14 @@
     <PackageReference Update="Microsoft.Azure.Test.HttpRecorder" Version="[1.13.3, 2.0.0)" />
     <PackageReference Update="Microsoft.Azure.WebJobs.Extensions" Version="4.0.1" />
     <PackageReference Update="Microsoft.Azure.WebJobs.Extensions.Http" Version="3.0.2" />
-    <PackageReference Update="Microsoft.CSharp" Version="4.6" />
+    <PackageReference Update="Microsoft.CSharp" Version="4.7" />
```

Comment:
Nit: might be equivalent, but should go with the full semver to avoid ambiguity i.e. 4.7.0.